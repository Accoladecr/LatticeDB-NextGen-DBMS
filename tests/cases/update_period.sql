CREATE TABLE events (
                        id TEXT PRIMARY KEY,
                        kind TEXT MERGE lww
);
INSERT INTO events (id, kind) VALUES ('e1','open');
UPDATE events SET kind='closed' VALID PERIOD ['2025-01-01T00:00:00Z','2025-02-01T00:00:00Z') WHERE id='e1';
SELECT id FROM events;
EXIT;
