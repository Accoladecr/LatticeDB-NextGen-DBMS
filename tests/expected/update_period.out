CREATE TABLE
INSERT 1 row(s)
UPDATE 1 row(s)
id
'e1'
Bye.
